hazelcast/src/main/java/com/hazelcast/internal/util/BiTuple.java

```diff
@@ -24,18 +24,18 @@
  * @param <Y> type of the second element
  */
 @SuppressWarnings("checkstyle:visibilitymodifier")
-public final class Tuple2<X, Y> {
+public final class BiTuple<X, Y> {
```

Comment:
wouldn't it be better to send this another PR, also including Tuple3 (which also exists in Jet?)

Reply:
Yes, it would be better, given the number of places affected. I send it in a separate PR and changing this to draft as long as the tuple PR is not merged.